var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -83,6 +83,9 @@ class Geant4(CMakePackage):
     variant("tbb", default=False, description="Use TBB as a tasking backend", when="@11:")
     variant("timemory", default=False, description="Use TiMemory for profiling", when="@9.5:")
     variant("vtk", default=False, description="Enable VTK support", when="@11:")
+    variant(
+        "data", default=True, sticky=True, description="Enable downloading of the data directory"
+    )
```

Comment:
Can we add some more documentation here, as comment, about how this is expected to be used when ~data? E.g. "user is expected to define GEANT4_INSTALL_DATADIR in their environment before running geant4". Maybe include an example that sets this to a cvmfs directory. Potentially also possible to write here some info about defining `geant4-data` (or individual datasets) as externals (we define g4ndl as external but install the rest).

Reply:
I've added a comment on this. Thanks!